lib/internal/child_process.js

```diff
@@ -230,12 +230,12 @@ util.inherits(ChildProcess, EventEmitter);
 
 
 function flushStdio(subprocess) {
-  if (subprocess.stdio == null) return;
-  subprocess.stdio.forEach(function(stream, fd, stdio) {
-    if (!stream || !stream.readable || stream._readableState.readableListening)
-      return;
-    stream.resume();
-  });
+  if (subprocess.stdio === null || subprocess.stdio === undefined) return;
+  for (let i = 0; i < subprocess.stdio.length; i++) {
+    if (!subprocess.stdio[i] || !subprocess.stdio[i].readable || subprocess.stdio[i]._readableState.readableListening) 
```

Comment:
Instead of looking up the value each time, add a temporary variable before the conditional. For example:

```js
const stdio = subprocess.stdio;
for (var i = 0; i < stdio.length; ++i) {
  const stream = stdio[i];
  // .... keep rest of the code the same
}
```